community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/cardinality/ExpressionSelectivityCalculator.scala

```diff
@@ -156,7 +157,8 @@ case class ExpressionSelectivityCalculator(stats: GraphStatistics, combiner: Sel
     val equality = java.math.BigDecimal.valueOf(calculateSelectivityForPropertyEquality(name, None, selections, propertyKeyName).factor)
     val factor = java.math.BigDecimal.valueOf(DEFAULT_RANGE_SEEK_FACTOR)
     val slack = BigDecimalCombiner.negate(equality).multiply(factor)
-    Selectivity(equality.add(slack).doubleValue())
+    val result = Selectivity(equality.add(slack).doubleValue())
+    result
```

Comment:
I tend to keep val result = ... in places that I find to debug often. This is one of them.